modules/contribute/pages/repositories.adoc

```diff
@@ -0,0 +1,30 @@
+= Repository Organization
+:url-git-sg:  https://github.com/stargate
+:url-git-sgdocs: {url-git-sg}/docs
+:url-git-sgantora-ui: {url-git-sg}/antora-ui-stargate
+:url-git-sgdocker: {url-git-sg}/docker-images
+
+//Stargate uses Antora to manage and build stargate.io/docs.
+
+The Stargate documentation is currently stored in a single repository that is
+organized in a standard directory structure. This may change in the future.
+
+[#repo-urls]
+// LLP 10.08.21 Not currently used, but when we break out the examples, we'll have
+// multiple repos.
+== Documentation Repository Roster
+
+The following repositories accept contributions via the regular workflow and use a standard branch pattern.
+
+* {url-git-sgdocs}[Stargate Docs^]
```

Comment:
Can we add the K8ssandra docs here, too?

Reply:
k8ssandra docs are not written in AsciiDoc, so these instructions don't apply to them.